bundles/org.eclipse.ui.ide/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Plugin.name
 Bundle-SymbolicName: org.eclipse.ui.ide; singleton:=true
-Bundle-Version: 3.21.200.qualifier
+Bundle-Version: 3.22.0.qualifier
```

Comment:
If we use newer API now, the version constraint on core resource should be changed, too, no?

Reply:
Good point, but why do we change **minor** version segment in first place? Shouldn't that be 3.21.300?

Reply:
Ok, I see, PR doesn't explain that. "Consume" sounds like "use it" but in fact, the public API in several classes has changed...

Reply:
Thanks for the reminder @szarnekow ! @robstryker can you please restrict the minimal version for the requirement on core.resources to ensure we link to a version that has the new API ?

Reply:
Done